splash/src/main/kotlin/edu/artic/splash/SplashActivity.kt

```diff
@@ -82,7 +82,7 @@ class SplashActivity : BaseViewModelActivity<SplashViewModel>(), TextureView.Sur
                             .setMessage(errorMessage)
                             .setPositiveButton(getString(android.R.string.ok)) { dialog, _ ->
                                 dialog.dismiss()
-                                if (it is NetworkErrorException) {
+                                if (it is NetworkException) {
```

Comment:
I do think we'll need a more extensive error-support in the near future. For now this will do.